fulp_modules/bloodsuckers/bloodsucker/bloodsucker_coffin.dm

```diff
@@ -0,0 +1,176 @@
+/datum/antagonist/bloodsucker/proc/ClaimCoffin(obj/structure/closet/crate/claimed) // NOTE: This can be any "closet" that you are resting AND inside of.
+	// ALREADY CLAIMED
+	if(claimed.resident)
+		if(claimed.resident == owner.current)
+			to_chat(owner, "This is your [src].")
+		else
+			to_chat(owner, "This [src] has already been claimed by another.")
+		return FALSE
+	// Bloodsucker Learns new Recipes!
+	owner.teach_crafting_recipe(/datum/crafting_recipe/vassalrack)
+	owner.teach_crafting_recipe(/datum/crafting_recipe/candelabrum)
+	owner.teach_crafting_recipe(/datum/crafting_recipe/meatcoffin)
```

Comment:
why aren't these taught to them when they initially become a bloodsucker? what if they want to prepare to gather the materials for these ahead of time?